templates/redmine/0/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2'
+
+services:
+  redmine-datavolume:
+    image: "busybox"
+    volumes:
+      - /home/redmine/data
+      - /var/lib/mysql
+      - /var/log/redmine
+    labels:
+      io.rancher.container.start_once: true
+    entrypoint: ["/bin/true"]
+
+  db:
```

Comment:
db shouldn't be a sidekick. Also, if you use "volumes_from", db would need another busybox just for him.